blog/models.py

```diff
@@ -25,3 +26,28 @@ class Meta:
         ordering = ['-added']
 
 
+
+class Comment(models.Model):
+    author = models.ForeignKey(settings.AUTH_USER_MODEL,
```

Comment:
Does this need to link back to the post rather? A comment pertains to a post, no?